exercises/leap/HINTS.md

```diff
@@ -0,0 +1,11 @@
+## Hints
+
+To complete this exercise you need to implement the function `isLeapYear`,
+that takes an year and returns if it is a leap year.
```

Comment:
I am sorry to mention this unfortunate ambiguity in English, but this makes me wonder if someone will read this sentence as the following: "returns if it is a leap year, and does not return (runs forever, etc.) if it is not a leap year". Hopefully nobody does (because such a function would not be very useful), but it could be.

We could be very explicit and say "returns a boolean indicating whether it is a leap year" or simply say "determines whether it is a leap year", perhaps?